### PR TITLE
Update manifest to version 2.3.1

### PR DESCRIPTION
**Pull request by jakobjakobson18**

### CODE REVIEW

**Comment by eszlari** [comment]
@jakobjakobson18 You can remove `org.audacityteam.Audacity.appdata.xml`, the upstream file (https://github.com/audacity/audacity/blob/master/help/audacity.appdata.xml) should work now. You just have to update the release entry with `sed` or `patch`.

**Comment by eszlari** [comment]
You need to add `"rename-appdata-file": "audacity.appdata.xml"`!

**Comment by jakobjakobson18** [comment]
> You need to add `"rename-appdata-file": "audacity.appdata.xml"`!

Thanks for the hint. I guess I could not have done it without it.

File: org.audacityteam.Audacity.yaml [hunk 1]
```diff
@@ -164,18 +165,10 @@ modules:
         aarch64:
           config-opts:
             - --disable-sse
-    post-install:
-      - chrpath -d /app/bin/audacity
-      - mv /app/share/mime/packages/audacity.xml /app/share/mime/packages/org.audacityteam.Audacity.xml
```
Comment (eszlari) [inline org.audacityteam.Audacity.yaml]: Please don't delete that line.

Comment (jakobjakobson18) [reply org.audacityteam.Audacity.yaml]: Okay

**Pull request merged**